python/chronos/src/bigdl/chronos/metric/forecast_metrics.py

```diff
@@ -15,10 +15,22 @@
 #
 
 import torch
+from torch import Tensor
 from numpy import ndarray
 from functools import partial
 from torchmetrics.functional import mean_squared_error, mean_absolute_error,\
-    mean_absolute_percentage_error, r2_score, symmetric_mean_absolute_percentage_error
+    mean_absolute_percentage_error, r2_score
+
+
+EPSILON = 1e-10
+
+
```

Comment:
add some comments here to tell future developers why we need to implement this smape ourselves.

Reply:
fix